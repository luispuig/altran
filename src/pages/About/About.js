// @flow
import React from 'react'

type _Props = {};

class About extends React.Component<_Props> {
  render() {
    return (
        <div>About Component!</div>
    );
  }
}

export default About
