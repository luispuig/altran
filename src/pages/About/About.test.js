import React from 'react';
import ReactDOM from 'react-dom';

import ShallowRenderer from 'react-test-renderer/shallow'; // ES6

import About from './About';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<About />, div);
});

it('match on Shallow mode', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<About />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
})

