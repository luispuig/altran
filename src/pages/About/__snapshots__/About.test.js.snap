// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match on Shallow mode 1`] = `
<div>
  About Component!
</div>
`;
